src/pykeen/trackers/base.py

```diff
@@ -50,8 +50,7 @@ def end_run(self, success: bool = True) -> None:
 
 
 class PythonResultTracker(ResultTracker):
-    """
-    A tracker which stores everything in Python dictionaries.
+    """A tracker which stores everything in Python dictionaries.
```

Comment:
😮

Reply:
`pykeen readme` does not like empty first lines 😕

Reply:
Haha this is obviously my doing. We could add a string strip in there so you can continue to write docstrings this way